### PR TITLE
[NO QA] [HOLD] Add instruction on setting up Mapbox credentials

### PR DESCRIPTION
**Pull request by hayata-suenaga**
### Details

We added a new library that uses native iOS and Android Mapbox SDKs. Mapbox is the service that offers, well, maps along with other map-related services. One caveat of using their SDKs is that they're closed-sourced, and you need a token to download them.

We created scripts `setup-mapbox-sdks` and `setup-mapbox-sdks-walkthrough` for streamline the process of obtaining the token and saving the token in the local environment. These scripts are invoked when running `npm run configure-mapbox`.

However, this script might not work for everyone, especially those developing on Windows computers. This PR adds instructions on manually obtaining and saving tokens.

### Fixed Issues

$ https://github.com/Expensify/App/issues/25452
PROPOSAL: N/A

### Test

This is a change in a documentation. No test or QA is needed.

cc: @neil-marcellini

### CODE REVIEW

**Comment by abdulrahuman5196** [comment]
@hayata-suenaga I have been assigned the auto C+ here? What should we do as C+ here?

**Review by allroundexperts — APPROVED** [review]

**Review by neil-marcellini — CHANGES_REQUESTED** [review]
Cool thanks, some small things

**Comment by neil-marcellini** [comment]
Also merge conflicts

File: README.md [hunk 1]
```diff
@@ -50,14 +50,14 @@ For an M1 Mac, read this [SO](https://stackoverflow.com/c/expensify/questions/11
 * Install project gems, including cocoapods, using bundler to ensure everyone uses the same versions. In the project root, run: `bundle install`
     * If you get the error `Could not find 'bundler'`, install the bundler gem first: `gem install bundler` and try again.
     * If you are using MacOS and get the error `Gem::FilePermissionError` when trying to install the bundler gem, you're likely using system Ruby, which requires administrator permission to modify. To get around this, install another version of Ruby with a version manager like [rbenv](https://github.com/rbenv/rbenv#installation).
-* Before installing iOS dependencies, you need to obtain a token from Mapbox to download their SDKs. Please run `npm run configure-mapbox` and follow the instructions.
+* Before installing iOS dependencies, make sure that you have setup Mapbox SDK credentials. Please refer to [Setting up Mapbox SDK Credentials](#setting-up-mapbox-sdk-credentials).
```
[the remark this answers is not in the document]
Comment (hayata-suenaga) [reply README.md]: thank you for thorough review 🙇

File: README.md [hunk 2]
```diff
@@ -50,14 +50,14 @@ For an M1 Mac, read this [SO](https://stackoverflow.com/c/expensify/questions/11
...
 * To install the iOS dependencies, run: `npm install && npm run pod-install`
 * If you are an Expensify employee and want to point the emulator to your local VM, follow [this](https://stackoverflow.com/c/expensify/questions/7699)
 * To run a on a **Development Simulator**: `npm run ios`
 * Changes applied to Javascript will be applied automatically, any changes to native code will require a recompile
 
 ## Running the Android app 🤖
-* Before installing Android dependencies, you need to obtain a token from Mapbox to download their SDKs. Please run `npm run configure-mapbox` and follow the instructions. If you already did this step for iOS, there is no need to repeat this step.
+* Before installing iOS dependencies, make sure that you have setup Mapbox SDK credentials. Please refer to [Setting up Mapbox SDK Credentials](#setting-up-mapbox-sdk-credentials).
```
[the remark this answers is not in the document]
Comment (hayata-suenaga) [reply README.md]: I changed iOS -> CocoaPods and Android -> Gradle

**Review by deetergp — DISMISSED** [review]

**Comment by deetergp** [comment]
You've got more conflicts, I'm afraid.

**Comment by hayata-suenaga** [comment]
I have to change the instruction in README because of this [issue](https://github.com/Expensify/Expensify/issues/309451). I added HOLD so that this PR cannot be merged.

**Comment by abdulrahuman5196** [comment]
@hayata-suenaga Is this PR still needed?

**Comment by hayata-suenaga** [comment]
yes this PR is still needed but we're holding this issue until the end of this month as this touches a build system (and there are new features merged evertyday until a big conference at the end of this month

**Comment by abdulrahuman5196** [comment]
Thank you for confirming. I am just checking through my K2 for pending items.

**Comment by hayata-suenaga** [comment]
Closing the PR as we're not deploying Mapbox script PR

**Pull request closed without merging**